www/src/components/repository/InstallDropdownButton.js

```diff
@@ -206,6 +203,9 @@ function InstallDropdownButton({ recipes, ...props }) {
             setRecipe(event.target.value)
             setTab(0)
           }}
+          endIcon={(
```

Comment:
Aren't we losing the original animation and arrow up icon while the dropdown is open?

Reply:
Yes, but that can be fixed later in honorable